kolibri/core/assets/src/views/language-switcher/LanguageSwitcherModal.vue

```diff
@@ -53,4 +82,24 @@
 </script>
 
 
-<style lang="scss" scoped></style>
+<style lang="scss" scoped>
+
+  @media only screen and (min-width: 840px) {
+    .language-column {
+      display: inline-block;
+      width: 50%;
+      margin: 20px 0;
+      &:nth-child(2) {
+        padding-left: 20px;
+      }
+    }
+  }
```

Comment:
We've essentially [banned css media queries](https://kolibri-dev.readthedocs.io/en/develop/references/conventions.html), and instead use this [responsive window](https://github.com/learningequality/kolibri/blob/develop/kolibri/core/assets/src/mixins/responsive-window.js) mixin throughout the codebase.

I think also that by making each column its own grid rather than using a single grid with internal columns, we're not leveraging the built-in responsive behaviors fully.